tests/scripts/gen_release_notes.sh

```diff
@@ -3,32 +3,14 @@ set -e
 
 function help() {
   print="
-  suppose we want release note for v1.6.8, we'll check the list of pr merged in upstream/release-1.6 but not present in v1.6.7
-  EXAMPLE command:
-  ./tests/scripts/gen_release_notes.sh <rook/rook remote name>/release-1.6 v1.6.7
+  To run this command,
```

Comment:
we don't need command line help if running from the github action, right?

Reply:
yes